cfme/tests/test_appliance.py

```diff
@@ -139,7 +139,7 @@ def test_db_connection(db):
     assert len(databases) > 0
 
 
-@pytest.mark.bugzilla(1121202)
+@pytest.mark.meta(blockers=["BZ#1121202"])
```

Comment:
Because I made this one before I started to mass-change all of them so ... the functionality is the same though. I'll change it if you want.